ddp-workspace/projects/ddp-rgp/src/assets/i18n/en.json

```diff
@@ -533,31 +540,38 @@
+                        "We will mail you a pre-paid specimen collection kit for you to have your blood drawn at a location of your choosing. We have partnered with Quest Diagnostics, a company with <a href=\"https://appointment.questdiagnostics.com/patient/confirmation\" target=\"_blank\" class=\"Link\">thousands of blood draw locations</a> across the United States and Puerto Rico, who will draw blood for the study and bill us directly. Alternatively, you can have your blood drawn by a local provider. If you are billed for this, send us a receipt for reimbursement.",
+                        "Regardless of where you have blood drawn, <span class=\"AccentUnderline\">please confirm with the technician that they have labeled both the blood tube and the paperwork with your legal name and date of birth</span>."
                     ]
                 },
                 {
                     "question": "What happens after I return my blood kit?",
                     "answers": [
-                        "After your kit returns to our facilities, we will extract DNA from your blood sample and prepare it for DNA sequencing or targeted testing."
+                        "After your kit is sent back to us, we will extract DNA from the blood sample and prepare it for DNA sequencing or targeted testing."
                     ]
                 },
                 {
                     "question": "Who in my family will get their genome sequenced?",
                     "answers": [
-                        "It depends. Typically, we perform genome sequencing on the affected individual(s) in the family and their biological parents (when available). We typically only perform targeted testing for specific variants of interest in unaffected family members."
+                        "It depends. Typically, we perform genome sequencing on the affected individual(s) in the family and their biological parents, when possible. We typically only perform targeted testing for specific variants of interest in unaffected family members."
+                    ]
+                },
+                {
+                    "question": "What medical records are needed?",
+                    "answers": [
+                        "We require medical records from the affected family member(s). For more information about the types of records that are needed, please see our <a href=\"/sharing-guide.pdf\" target=\"_blank\" class=\"Link\">Guide to Sharing Health Information</a>."
```

Comment:
Somehow this link to the sharing-guide PDF in the FAQ page is not working for me when I run the app locally.

Reply:
Interesting, it doesn't work for me either

Reply:
For now, I think I'm going to merge this in as-in and investigate/fix it in a separate PR.